src/main/scala/com/gu/CheckPriceRisePreConditions.scala

```diff
@@ -0,0 +1,65 @@
+package com.gu
+
+import com.gu.FileImporter.PriceRise
+
+import scala.util.Try
+
+trait PriceRisePreCondition
+case object SubscriptionIsAutoRenewable extends PriceRisePreCondition
+case object SubscriptionIsActive extends PriceRisePreCondition
+case object DeliveryRegionMatchesCurrency extends PriceRisePreCondition
```

Comment:
I'm not sure about this condition, for example I think it's valid for a RoW subscriber to pay in GBP _or_ USD.